sklearn/preprocessing/data.py

```diff
@@ -210,6 +210,11 @@ class MinMaxScaler(BaseEstimator, TransformerMixin):
 
     where min, max = feature_range.
 
+    The transformation is calculated as::
```

Comment:
This seems to be talking about transformation of the training data? Shouldn't these formulas include the _ attributes?

Reply:
Yes, the goal is to help users understand the algorithm (users can easily extend it to new data)
I don't use _ attributes to keep consistent with the way we define the transformation:
```
The transformation is given by::
        X_std = (X - X.min(axis=0)) / (X.max(axis=0) - X.min(axis=0))
        X_scaled = X_std * (max - min) + min
    where min, max = feature_range.``
```

Reply:
> Yes, I think that is more accessible

So what's your suggestion here?